buildpacks/ruby/src/layers/bundle_download_layer.rs

```diff
@@ -82,20 +83,21 @@ pub(crate) struct MetadataV1 {
     pub(crate) version: ResolvedBundlerVersion,
 }
 
+#[tracing::instrument(skip(bullet, env, gem_path), err)]
```

Comment:
I'm curious what the result of including `err` in the trace here will provide. I mean, apart from having the error attached directly to this span. Because of how error handling in the CNBs works, you may want to consider centralizing the capturing of errors to the `on_error` handler. 

I don't consider this a problem that needs to be fixed, it's just a different approach than I was expecting to see.

Reply:
I copied this from the go buildpack originally. I'll just disable all of them.